doc/supported_operations.md

```diff
@@ -0,0 +1,16 @@
+# Containerized Data Importer supported operations
+The Containerized Data Importer (CDI) supports importing data/disk images from a variety of sources, and it can be hard to understand which combination of operations is supported and if that operation requires scratch space. This document explains the combinations that are supported and which ones require scratch space. All Kubevirt content type operations will support both file system Data Volumes (DV) as well as block volume DVs.
+
+## Supported matrix
+
+| Content-Type | http | https | http basic auth | Registry | Upload | S3 Bucket  |
+|--------------|---------|-|--|-------|--------|------------|
+| KubeVirt(QCOW2)        |<ul><li>[x] QCOW2</li><li>[x] GZ\*</li><li>[x] XZ\*</li><li>[x] TAR\*</li></ul> |<ul><li>[x] QCOW2\*\*</li><li>[x] GZ\*</li><li>[x] XZ\*</li><li>[x] TAR\*</li></ul> |<ul><li>[x] QCOW2</li><li>[x] GZ\*</li><li>[x] XZ\*</li><li>[x] TAR\*</li></ul> | <ul><li>[ ] QCOW2</li><li>[ ] GZ</li><li>[ ] XZ</li><li>[ ] TAR</li></ul> | <ul><li>[x] QCOW2\*</li><li>[x] GZ\*</li><li>[x] XZ\*</li><li>[x] TAR\*</li></ul> | <ul><li>[x] QCOW2\*</li><li>[x] GZ\*</li><li>[x] XZ\*</li><li>[x] TAR\*</li></ul> |
```

Comment:
I think we need another * to remind users that qcow2 files are converted to raw in order to avoid confusion there.

Reply:
Hopefully explained that in the new paragraph that explains the first column of the grid.